.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,12 @@ https://github.com/nimblehq/git-template/issues/??
 
 ## What happened 👀
 
-Describe the big picture of your changes here to communicate to the team why we should accept this pull request. 
- 
+Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
+
 ## Insight 📝
 
-Describe in details how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
- 
+Describe in detail how to test the changes, which solution you tried but did not go with, referenced documentation is welcome as well.
+
 ## Proof Of Work 📹
 
-Show us the implementation: screenshots, gif, etc.
+Show us the implementation: screenshots, GIT, etc.
```

Comment:
@malparty Should it be `GIF` 😄

Reply:
Nice catch! >< 
10ead89 

I've also pluralize GIFs as suggested by Thuumb in the `git-templates` repository~